x-pack/plugins/security_solution/public/common/lib/telemetry/events/document_details/types.ts

```diff
@@ -11,6 +11,7 @@ import type { TelemetryEventTypes } from '../../constants';
 export interface ReportDetailsFlyoutOpenedParams {
   location: string;
   panel: 'left' | 'right' | 'preview';
+  // key: string;
```

Comment:
@PhilippeOberti I thought about adding a `key` param to track what kind of panel is opened. It can give us better insights of what people are previewing, caveats:
- This impacts all the existing open flyout telemetry
- I created an issue (https://github.com/elastic/security-team/issues/9964) to standardize all the flyout telemetry, which will likely remove all the telemetry in components. Given the timing, not sure it makes sense to add complexity now and later remove them in 8.16..?
Wdyt?

Reply:
I had a similar idea when I added last some telemetry code, but decided against it as I wasn't sure it was very useful for what we added telemetry in the first place. At the time, we just wanted to know if the expandable flyout was used.
I wonder if we could achieve the same output by just looking at url, as all the information would be in there...

Reply:
@PhilippeOberti yeah i agree. at this point, i think any telemetry is better than no telemetry. are you ok with merging this pr (adding basic preview telemetry), and we discuss enhancements in 8.16?

Reply:
Yes let me approve it right away!